remove_dimension.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+from numpy import array
+
+
+def remove_dimension(matrix: array) -> array:
+    """
+    The function removes a single-dimensional entries from a specified shape.
+    :param matrix: A given matrix.
+    :return: The matrix with less dimensional entry.
```

Comment:
Are you returning the matric or it's shape?

Reply:
Fixed. (removed ".shape")